app/api/auth/create_access_token.go

```diff
@@ -122,7 +131,12 @@ const parsedRequestData ctxKey = iota
 //			type: integer
 //			enum: [0,1]
 //			default: 0
+//		- name: create_temp_user_if_not_authorized
+//			description: Whether to create a temporary user if the token is not provided or expired.
+//			in: query
+//			type: boolean
```

Comment:
may be useful to mention that if not given, it is assumed `false`

Reply:
fixed, added `default`. Also changed type `boolean` to `integer` with enum `[0, 1]`, because that's what we want.